sycl/test-e2e/KernelAndProgram/free_function_apis.cpp

```diff
@@ -207,6 +208,38 @@ bool test_bundle_apis(queue Queue) {
   std::cout << "PassR=" << PassR << std::endl;
   Pass &= PassR;
 
+  kernel_bundle Bundle_ff2 = ext::oneapi::experimental::get_kernel_bundle<
+      ff_2, bundle_state::executable>(Context);
+
+  bool PassS =
+      ext::oneapi::experimental::get_kernel_info<ff_2,
+                                                 info::kernel::function_name>(
+          Context) == Bundle_ff2.ext_oneapi_get_kernel<ff_2>()
+                          .get_info<info::kernel::function_name>();
+  std::cout << "PassS=" << PassS << std::endl;
+  Pass &= PassS;
+
+  kernel_bundle Bundle_ff3 = ext::oneapi::experimental::get_kernel_bundle<
+      ff_3<int>, bundle_state::executable>(Context);
+
+  bool PassT =
+      ext::oneapi::experimental::get_kernel_info<
+          ff_3<int>, info::kernel_device_specific::work_group_size>(Context,
+                                                                    Device) ==
+      Bundle_ff3.ext_oneapi_get_kernel<ff_3<int>>()
+          .get_info<info::kernel_device_specific::work_group_size>(Device);
+  std::cout << "PassT=" << PassT << std::endl;
+  Pass &= PassT;
+
+  bool PassU =
+      ext::oneapi::experimental::get_kernel_info<
+          ff_3<int>, info::kernel_device_specific::work_group_size>(Queue) ==
+      Bundle_ff3.ext_oneapi_get_kernel<ff_3<int>>()
+          .get_info<info::kernel_device_specific::work_group_size>(
+              Queue.get_device());
+  std::cout << "PassU=" << PassU << std::endl;
```

Comment:
NIT: I don't really like `std::cout << "PassU=" ...`, instead of printing `PassU=`, can we be more descriptive, like `cout << "Test retrieving WGSize: "`?